index.css

```diff
@@ -88,3 +90,194 @@ body {
 .social:hover {
   cursor: pointer;
 }
+
+.recent-title {
+  font-family: 'Crete Round', serif;
+  font-size: 2.5rem;
+  font-weight: normal;
+  text-align: center;
+  margin: 7.5rem 0 0 0;
+}
+
+hr {
+  margin: 0 2rem;
+  margin-top: 2rem;
+  margin-bottom: 5rem;
+}
+
+.work-section {
+  display: grid;
+  grid-template-columns: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  margin: 1rem 2rem;
+}
+
+.first-project {
+  min-width: 337px;
+  height: 237px;
+}
+
+.project-title {
+  font-size: 2rem;
+  font-family: 'Crete Round', serif;
+  font-weight: normal;
+  font-weight: 500;
+}
+
+.used-tools {
+  list-style-type: none;
+  display: flex;
+  justify-content: start;
+  align-items: start;
+  margin: 0.5rem 0 1.1rem 0;
+  padding: 0;
+}
+
+.used-tools > li {
+  font-size: 1rem;
+  padding: 10px 12px;
+  border: 1px solid #8993a4;
+  margin-right: 1rem;
+}
+
+.project-description {
+  font-family: 'Inter', sans-serif;
+  font-size: 1rem;
+  margin-top: -0.5rem;
+}
+
+.see-project-btn {
+  font-family: 'Inter', sans-serif;
+  font-weight: 600;
+  font-size: 1rem;
+  background-color: #ff6b00;
+  border: none;
+  color: #fff;
+  width: 7.75rem;
+  height: 3rem;
+}
+
+.see-project-btn:hover {
+  color: #f55800;
+  background-color: #fff5e1;
+  border: 1px solid #f55800;
+}
+
+.see-project-btn:active {
+  color: #f4f1f0;
+  background-color: #e05304;
+  border: none;
+}
+
+.not-featured {
+  background-image: url('./images/project_bg.png');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+  margin-bottom: 2rem;
+}
+
+.not-featured-title {
+  padding-top: 4rem;
+  color: #fff;
+  margin-left: 1rem;
+}
+
+.not-featured-desc {
+  color: #fff;
+  margin-left: 1rem;
+  margin-right: 1rem;
+}
+
+.tools-bottom {
+  margin-left: 1rem;
+  color: #fff;
+}
+
+.tools-bottom > li {
+  background-color: rgba(255, 255, 255, 0.24);
+  border: none;
+}
+
+.full-width-btn {
+  width: 100%;
+}
+
+.about-me {
+  background-color: #1c1a19;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  margin: 5.5rem 1.5rem;
+  padding-bottom: 7.5rem;
+  padding-top: 3rem;
+  background-image: url('./images/illustration-about-me-mobile.png');
```

Comment:
Hey 👋🏽 how about instead of adding that specific element from the Figma as a background image 🖼️  you used the background image containing this `section` instead? 🤔 This is just a suggestion because I noticed that your design still looks the same despite the recommendations of the previous reviewer.

## Your design
![Screenshot 2023-01-14 at 07 38 46](https://user-images.githubusercontent.com/45739963/212471907-19327074-9ba7-47d0-b3db-03e44133fbc2.png)

## Figma Design
![Screenshot 2023-01-14 at 07 39 23](https://user-images.githubusercontent.com/45739963/212471925-1e0f3052-05a2-427a-9105-ebc3012e500a.png)

Reply:
Hello @didierganthier ,
Thank you for the feedback.
I did not understand how this is happening. As you can see I have updated the codebase and this is what I'm getting when I try it
![about-me](https://user-images.githubusercontent.com/77828236/212473046-5917c239-c3f1-43f0-89ea-824281031ac8.png)

Would you please check it one more time maybe the remote origin was not updated. 
Thank you so much